src/NHibernate/NHibernate.csproj

```diff
@@ -1,42 +1,72 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <Import Project="../../build-common/NHibernate.props" />
+
   <PropertyGroup>
-    <Description>An object persistence library for relational databases.</Description>
+    <Description>NHibernate is a mature, open source object-relational mapper for the .NET framework. It is actively developed, fully featured and used in thousands of successful projects.</Description>
```

Comment:
Please put this into `<PackageDescription>`

Reply:
@hazzik  what are you referencing?  `<Description>` is what Visual Studio fills out based on the project package property page, and what is documented here: https://docs.microsoft.com/en-us/dotnet/core/tools/csproj#description

Reply:
I don't care about this documentation. I'm talking about this one: https://github.com/NuGet/NuGet.Client/blob/dev/src/NuGet.Core/NuGet.Build.Tasks.Pack/Pack.targets#L28

Reply:
I agree that the naming of properties intended to be set in `.csproj` for generating `.nuget` packages are not altogether consistent, but they do work as documented.

If you set the description in `<PackageDescription>`, it may override that pack.targets property, but it no longer shows up in Visual Studio's project property page for the Package description, so `<PackageDescription>` clearly is not the property that is intended to be set in the `.csproj` file.  The line you reference is by default set by `$(Description)`, which we were already setting in the `.csproj`. I just adjusted the text so the output `.nuspec` matches the old templated one.

If you compile and compare the extracted `.nuget` packages, it all works fine.

Reply:
The problem that `Description` also meant to set `AssemblyDescriptionAttribute`. The `PackageDescription` is what transferred into the `PackTask` (L211). So, it is a correct way to set the separate descriptions for a package and an assembly.

Reply:
ok... but where does that ever show up in normal use?  It doesn't show up in the `NHibernate.dll` file Properties dialog.

Is there a problem if the text on `AssemblyDescriptionAttribute` does match the NuGet package description?

Reply:
https://github.com/NuGet/NuGet.Client/pull/1648

Reply:
I'm not saying that they don't collide, obviously they do.  I was asking why it matters?  Also, the NuGet/NuGet.Client#1648 pull request was only 2 months ago, so it wasn't included in the .net core 2.0 release.  On my computer, the file at:

`C:\Program Files\dotnet\sdk\2.0.0\Sdks\NuGet.Build.Tasks.Pack\build\NuGet.Build.Tasks.Pack.targets` doesn't have `<PackageDescription>` and instead relies on `<Description>`:

```xml
    <Description Condition="'$(Description)'==''">Package Description</Description>
```

Reply:
It's in 2.0.2